From 47f215ce18ef17c6748c4fcc94e8b7c0245ac2b7 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Wed, 29 Jul 2015 21:25:37 +0200
Subject: [PATCH 2/5] fix warning when __GLIBC__ is not defined

__GLIBC__ must be defined before testing the glibc version
with (__GLIBC__ <= 2 && __GLIBC_MINOR__ <= 2).

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 src/daemon/caps.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/caps.c b/src/daemon/caps.c
index 669d6e2..a9cf7d4 100644
--- a/src/daemon/caps.c
+++ b/src/daemon/caps.c
@@ -39,7 +39,7 @@
 #include "caps.h"
 
 /* Glibc <= 2.2 has broken unistd.h */
-#if defined(__linux__) && (__GLIBC__ <= 2 && __GLIBC_MINOR__ <= 2)
+#if defined(__linux__) && defined(__GLIBC__) && (__GLIBC__ <= 2 && __GLIBC_MINOR__ <= 2)
 int setresgid(gid_t r, gid_t e, gid_t s);
 int setresuid(uid_t r, uid_t e, uid_t s);
 #endif
-- 
2.4.3

